libykclient.c

```diff
@@ -213,7 +213,7 @@ yubikey_client_request (yubikey_client_t client,
 {
   struct MemoryStruct chunk = { NULL, 0 };
   yubiauth_dir_cfg *cfg = ap_get_module_config(r->per_dir_config, &authn_yubikey_module);
-  const char *url_template = "%s://%s/wsapi/verify?id=%d&otp=%s";
+  const char *url_template = "%s://%s/wsapi/2.0/verify?id=%d&otp=%s&nonce=1234567890abcdef";
```

Comment:
Can we generate a simple `nonce` here, rather than this static string?

Reply:
As it turns out the nonce is not used on the client side at all -- so the strength of the random nonce being generated really doesn't matter

Until the libykclient.c is fully replace with ykclient.c I think the above fix is equal in terms as of security as the current code base